<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>524</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>524</width>
    <height>600</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color :#3a3a3a;
color : #797979;</string>
  </property>
  <widget class="QLabel" name="labelPictureNolt">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>115</y>
     <width>352</width>
     <height>104</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>48000</width>
     <height>48000</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/new/login/images/loginBackground.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>419</y>
     <width>125</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Proxima Nova Lt</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid #d0d0d0;
color : #d0d0d0;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string>LOG IN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLogin">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>327</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Proxima Nova Th</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-bottom:1px solid #797979;
color : #d0d0d0;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>373</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Proxima Nova Th</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-bottom:1px solid #797979;
color : #d0d0d0;
background-color:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogin">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>332</y>
     <width>120</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Proxima Nova Th</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color : #797979;
background-color:transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>378</y>
     <width>120</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Proxima Nova Th</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color : #797979;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIncorrectLogin">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>465</y>
     <width>340</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Proxima Nova Th</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : #797979;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Login and password is not correct</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPictureCat">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>524</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524</width>
     <height>600</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/new/login/images/loginFormCat.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSimpleLine">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>437</y>
     <width>59</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-top: 1px solid #0992b1;
background:transparent;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/login/images/closeIcon.png</normaloff>:/new/login/images/closeIcon.png</iconset>
   </property>
  </widget>
  <zorder>labelPictureCat</zorder>
  <zorder>labelPictureNolt</zorder>
  <zorder>btnLogin</zorder>
  <zorder>lineEditLogin</zorder>
  <zorder>lineEditPassword</zorder>
  <zorder>labelLogin</zorder>
  <zorder>labelPassword</zorder>
  <zorder>labelIncorrectLogin</zorder>
  <zorder>lineEditSimpleLine</zorder>
  <zorder>btnQuit</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
